NitroxClient/MonoBehaviours/MultiplayerButton.cs

```diff
@@ -0,0 +1,234 @@
+﻿using NitroxModel.Helper;
+using NitroxClient.MonoBehaviours;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+using UWE;
+using NitroxClient.Communication;
+
+namespace NitroxClient.MonoBehaviours
+{
+    public class MultiplayerButton : MonoBehaviour
+    {
+        public void Awake()
+        {
+            MultiplayerMenuMods();
+        }
+
+        private void MultiplayerMenuMods()
+        {
+            GameObject startButton = GameObject.Find("Menu canvas/Panel/MainMenu/PrimaryOptions/MenuButtons/ButtonPlay");
+            GameObject showLoadedMultiplayer = Instantiate(startButton);
+            Text buttonText = showLoadedMultiplayer.transform.Find("Circle/Bar/Text").gameObject.GetComponent<Text>() as Text;
```

Comment:
minor: do we need to say "as Text" - it looks like you are already using the generic method call in get component;